### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by amandahla**
Applicable spec: <link>

### Overview

Update README according to the new template.

### Rationale

Follow the new template pattern.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,31 +20,48 @@ straightforward through Juju’s clean interface. It will allow easy deployment
...
+### Basic operations
+
+#### Add plugins and custom images
+Refer to [How to add plugins and custom images](https://charmhub.io/mattermost-k8s/docs/add-plugins-custom-images)
+for step-by-step instructions.
+
+#### Authenticate
+Refer to [How to authenticate](https://charmhub.io/mattermost-k8s/docs/authenticate)
+for step-by-step instructions.
+
```
Comment (erinecon) [inline README.md:57]: Are there any required integrations for the Mattermost charm to work? If so, then I would add a `## Integrations` section and mention them. If you do add an Integrations section, then I would also link the [Integrations](https://charmhub.io/mattermost-k8s/integrations) page of the Charmhub docs.

Comment (amandahla) [reply README.md:57]: Yes, PostgreSQL is a required integration, good point. Integrations section added!

File: README.md [hunk 2]
```diff
@@ -20,31 +20,57 @@ straightforward through Juju’s clean interface. It will allow easy deployment
...
+## Integrations
+
+- [Postgresql](https://charmhub.io/postgresql-k8s) (required): PostgreSQL is a
```
[the remark this answers is not in the document]
Comment (amandahla) [reply README.md]: Fixed, thanks

**Review by erinecon — APPROVED** [review]
LGTM :+1:

**Review by javierdelapuente — APPROVED** [review]

**Pull request merged**